verify-merge.py

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env python3
+import argparse
+import os
+import subprocess
+import glob
+
+GIT = os.getenv('GIT','git')
+GPG = os.getenv('GPG','gpg')
+
+def verify():
+    global args, workdir
+    os.chdir('gitian-pubkeys')
+    print('Importing pubkeys...')
+    keys = [f for f in glob.glob("*.asc", recursive=True)]
+    for key in keys:
+        subprocess.check_call([GPG, '--import', key])
+    print('Refreshing pubkeys...')
+    subprocess.check_call([GPG, '--refresh'])
+    os.chdir('../../gitian-builder')
+    print('\nVerifying '+args.version+' Linux\n')
+    subprocess.check_call(['bin/gverify', '-v', '-d', '../gitian.sigs/', '-r', args.version+'-linux', '../monero/contrib/gitian/gitian-linux.yml'])
```

Comment:
Hard-coding the folder name `gitian.sigs` means it will break if the user cloned it into a different folder (as I did).  Would be better to get the directory name from something like `os.getcwd().split(os.sep)[-1]` at the beginning of the script.  This will provide the working directory from which the script is being run.